### PR TITLE
chore: always using 0.0.0 for helm chart in repo

### PR DESCRIPTION
**Pull request by STRRL**
Signed-off-by: STRRL <im@strrl.dev>

### What problem does this PR solve?

close https://github.com/chaos-mesh/chaos-mesh/issues/3310

### What's changed and how it works?

- change `version` and `appVersion` in helm charts to `0.0.1`

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [ ] release-2.2
  - [ ] release-2.1

### Checklist

CHANGELOG

- [x] I have updated the `CHANGELOG.md`
- [ ] I have labeled this PR with "no-need-update-changelog"

Tests

- [ ] Unit test
- [ ] E2E test
- [x] No code
- [ ] Manual test (add steps below)

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

**Comment by STRRL** [comment]
/cc @YangKeao @Hexilee @cwen0 

PTAL

File: helm/chaos-mesh/Chart.yaml [hunk 1]
```diff
@@ -13,14 +13,14 @@
 # limitations under the License.
 #
 apiVersion: v2
-appVersion: "v0.9.0"
+appVersion: "0.0.1"
```
Comment (YangKeao) [inline helm/chaos-mesh/Chart.yaml:16]: IIUC, the `appVersion` will always be replaced by a version specified in the command line? (as commented in https://github.com/chaos-mesh/chaos-mesh/issues/3274#issuecomment-1135547412 ) Then what about using an obviously special version like "0.0.0" or "9999"...

"0.0.1" sounds like an available version (smaller than everyone else)

Comment (STRRL) [reply helm/chaos-mesh/Chart.yaml]: sounds great! `0.0.0` is also a valid sem ver. Updated!

**Comment by STRRL** [comment]
/cc @YangKeao 
PTAL

**Review by Hexilee — APPROVED** [review]
LGTM

**Review by YangKeao — APPROVED** [review]
LGTM

**Pull request merged**